rhino/src/main/java/org/mozilla/javascript/regexp/NativeRegExp.java

```diff
@@ -289,6 +296,246 @@ Object execSub(Context cx, Scriptable scopeObj, Object[] args, int matchType) {
         return rval;
     }
 
+    private static void prettyPrintRE(RECompiled regexp) {
```

Comment:
It looks like we don't have tests that exercise this. Is there a way we can put something in, even as a "smoke test" just to make sure that we call it and it doesn't bomb out immediately?

Reply:
Good point! Only way I could think of was to use reflection to call the pretty printer in a test. I've added such a test.